modules/migration-using-must-gather.adoc

```diff
@@ -73,12 +83,17 @@ endif::[]
 ifdef::oadp-troubleshooting[]
 * Prometheus metrics data dump:
 endif::[]
+.. For OADP 1.2 use the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ oc adm must-gather --image={must-gather} \
-  -- /usr/bin/gather_metrics_dump
+oc adm must-gather --image=registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2 -- /usr/bin/gather_metrics_dump
```

Comment:
What is the purpose of the c and d steps? Could a note be placed under steps a and b to mention a storage location? RHEL versions? Maybe this should be clarified?

Reply:
@PrasadJoshi12 
What are your thoughts?

Reply:
@dfitzmau 

Thank you for your comments. You are correct that this section could use some reworking, but that is beyond the scope of the current PR. I have opened a ticket requesting a future doc improvement for this section. 

https://issues.redhat.com/browse/OADP-3470

Thank you very much for all of your feedback.